src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/TempPE/ITempPEBuildManager.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.TempPE
+{
+    /// <summary>
+    /// Provides TempPE build information and services to the build manager
+    /// </summary>
+    internal interface ITempPEBuildManager
+    {
+        /// <summary>
+        /// Get the list of design time monikers that need to have TempPE libraries created
+        /// </summary>
+        string[] GetTempPEMonikers();
+
+        /// <summary>
+        /// Get the XML used by the type resolution service that describes the TempPE library for the specified input moniker, and triggers the compilation of the library if necessary
+        /// </summary>
+        /// <param name="moniker">On of the monikers return from <see cref="GetTempPEMonikers"/> for which a TempPE library should be produced.</param>
```

Comment:
>return [](start = 53, length = 6)

returned?

Reply:
Thanks. Looks like "On" should also be "One" :)